packages/trader/src/App/Components/Elements/PositionsDrawer/helpers/positions-helper.ts

```diff
@@ -27,7 +34,9 @@ export const getBarrierValue = (contract_info: TContractInfo) => {
     if (isDigitType(contract_info.contract_type)) {
         return digitTypeMap(contract_info)[contract_info.contract_type as keyof ReturnType<typeof digitTypeMap>];
     }
-    return addCommaToNumber(Number(contract_info.barrier));
+    return contract_info.barrier
+        ? formatMoney(contract_info.currency ?? 'USD', contract_info.barrier, true, 0, 0)
+        : null;
```

Comment:
```suggestion
        : '';
```
@kate-deriv is it supposed to return a string? please check🙏🏻

Reply:
Previously it was returning `string | number | null | undefined` :)
<img width="878" alt="Screenshot 2023-11-16 at 15 15 31" src="https://github.com/binary-com/deriv-app/assets/121025168/3fddb703-6d7d-4cd8-9030-9bd26df069b5">
But anyway I can return an empty string if you want